src/ceph.in

```diff
@@ -849,7 +859,12 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, filter=True)
+
+    # implement "tell module help"
+    if childargs and len(childargs) >= 3 and childargs[0] == "tell" and childargs[2] == "help":
```

Comment:
Python's [doc](https://docs.python.org/2/library/argparse.html#argparse.ArgumentParser.parse_known_args) says that childargs will be [], not `None`.

I'll test it and update. but i prefer to take None check, it's more safe.